src/shared/components/query/MolecularProfileSelector.tsx

```diff
@@ -1,174 +1,174 @@
-import * as _ from 'lodash';
-import * as React from 'react';
-import {MolecularProfile} from "../../api/generated/CBioPortalAPI";
-import FontAwesome from "react-fontawesome";
-import * as styles_any from './styles/styles.module.scss';
-import {observer} from "mobx-react";
-import classNames from 'classnames';
-import { FlexRow } from "../flexbox/FlexBox";
-import {QueryStoreComponent} from "./QueryStore";
-import DefaultTooltip from "../../../public-lib/components/defaultTooltip/DefaultTooltip";

```

Comment:
This is actually the only change in this file. The whole file seems to be updated because of a `new line` difference between Windows and other systems.